app/sagas/login.registerPushTokenFork.test.ts

```
@@ -0,0 +1,102 @@
+	active: {
+		get: jest.fn()
+	}
+}));
+
+jest.mock('../lib/services/restApi', () => ({
+	registerPushToken: jest.fn().mockResolvedValue(undefined),
+	pendingToken: '',
+	pendingVoipToken: ''
+}));
+
+import { registerPushToken } from '../lib/services/restApi';
```

ESLint

[error] Import in body of module; reorder to top.

(import/first)